pages/emailSender.tsx

```diff
@@ -305,16 +334,40 @@ const EmailSender: NextPage = () => {
     )
   }
 
-  const sendEmails = () => {
+  const sendEmails = async () => {
```

Comment:
It might be good to display some kind of loading state while the file is being uploaded, whether that's some text that says "uploading file..." or a loading spinner or something else. Not blocking, can add next week!

Reply:
Yep agree! Adding it to my week 2 to do list:)